pages/edit_group.py

```diff
@@ -135,12 +173,20 @@ def name(self):
         class InviteForm(Region):
             _invite_search_locator = (By.CSS_SELECTOR, '.select2-search__field')
             _invite_locator = (By.ID, 'form-submit-invite')
+            _invites_list_locator = (By.CSS_SELECTOR, '.select2-selection__choice')
+
+            @property
+            def invites_list(self):
+                return self.find_elements(*self._invites_list_locator)
 
             def invite_new_member(self, mozillian):
                 search = self.find_element(*self._invite_search_locator)
                 search.send_keys(mozillian)
-                self.wait.until(expected.visibility_of_element_located((
-                    By.XPATH, '//li[contains(text(), "{0}")]'.format(mozillian)))).click()
+                try:
+                    self.wait.until(expected.visibility_of_element_located((
+                        By.XPATH, '//li[contains(text(), "{0}")]'.format(mozillian)))).click()
+                except Exception:
```

Comment:
It's not a good idea to use bare Exception catching. If the element not being visibile, and not clicking on it is an acceptable path then I think we need a separate method for this scenario. At best, this path will incur a delay while we wait for the element to be displayed. We may also miss failures for tests that are expecting this to perform the click.